scitt/check_operation_status.py

```diff
@@ -0,0 +1,46 @@
+""" Module for checking when a statement has been anchored in the append-only ledger """
+
+import json
+import subprocess
+import argparse
+import time
+
+
+def check_operation_id(
+    operation_id: str
+)-> str:
+
+    return subprocess.check_output("curl -s -H @$HOME/.datatrails/bearer-token.txt https://app.datatrails.ai/archivist/v1/publicscitt/operations/"+operation_id, shell = True).decode()
```

Comment:
Please use the requests library if you want to call a https endpoint.

Also include a timeout for the request so it doesn't run forever.

Reply:
Update in flight...

Reply:
updated